client/resdef.go

```diff
@@ -22,19 +22,31 @@ import (
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/api/v1"
 	"k8s.io/kubernetes/pkg/apis/batch"
+	"k8s.io/kubernetes/pkg/apis/extensions"
 	"k8s.io/kubernetes/pkg/client/restclient"
 )
 
+type ReplicaSetDemand struct {
```

Comment:
I'd call it just ReplicaSet. It gonna be client.ReplicaSet and it will differ from both extensions.ReplicaSet and resource.ReplicaSet